lib/mix/tasks/phoenix.gen.model.ex

```diff
@@ -70,7 +70,8 @@ defmodule Mix.Tasks.Phoenix.Gen.Model do
     ]
   end
 
-  defp validate_args!([_, plural | _] = args) do
+  defp validate_args!([singular, plural | _] = args) do
+    Mix.Phoenix.check_module_name_availability!(singular)
```

Comment:
@josevalim does the namespace need to be considered?  In issue #921, even within the `Throwaway` namespace an error is still raised:

```elixir
(UndefinedFunctionError) undefined function: Throwaway.Repo.start_link/0
```

Apologizes in advance if I'm confused :smile:, thanks for your guidance.